Makefile

```diff
@@ -33,6 +33,7 @@ GO_LD_FLAGS := -ldflags "\
 	-X github.com/kubev2v/migration-planner/pkg/version.commitFromGit=$(SOURCE_GIT_COMMIT) \
 	-X github.com/kubev2v/migration-planner/pkg/version.gitTreeState=$(SOURCE_GIT_TREE_STATE) \
 	-X github.com/kubev2v/migration-planner/pkg/version.buildDate=$(BIN_TIMESTAMP) \
+	-X github.com/kubev2v/migration-planner/internal/agent.version=$(SOURCE_GIT_TAG) \
```

Comment:
this will be empty until we start to release no? I would try to set version either to tag or git sha

Reply:
No it's SHA, when we release it will be tag.

Reply:
INFO[0000]/Users/omachace/work/migration-planner/pkg/log/log.go:49 github.com/kubev2v/migration-planner/pkg/log.(*PrefixLogger).Infof() Starting agent: c414cca